docs/guide/website/sidebars.json

```diff
@@ -1,8 +1,8 @@
 {
   "docs": {
     "Release Notes": ["releases/migrate", "updating"],
-    "Getting Started": ["introduction", "installation", "quickstart"],
-    "Main Concepts": [
+    "Getting Started": ["installation", "quickstart"],
```

Comment:
Let's keep the introduction. I don't think the "what is botpress, why botpress, etc." has its place in installation or quickstart.